.changeset/cuddly-glasses-battle.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/integration-react': patch
+---
+
+Remove unnecessary broad permissions from Gitlab SCMAuth
+
+Newer versions of Gitlab (after 2019) do not require the broad api permissions to write to repos.
```

Comment:
Could you please provide a link to the Gitlab documentation that lists the necessary permissions? This would be a good link to change announcement.

Reply:
So, [this MR](https://gitlab.com/gitlab-org/gitlab-foss/-/issues/58847) introduces the change and the [changelog for 11.11.0](https://gitlab.com/gitlab-org/gitlab/-/blob/a8325c425/CHANGELOG.md#added-36-changes-10-of-them-are-from-the-community) has the change included. These are both older than the code I changed, though...

Reply:
Thanks for sharing 🙌🏻